src/components/RankingAnyTagItem.tsx

```diff
@@ -1,22 +1,39 @@
-import styled from '@emotion/styled';
+import StyledRankingAnyTagItem from '@/styles/StyledRankingAnyTagItem';
+import { useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
 
 const anyTagItemlist = ['받고 싶어한', '많이 선물한', '위시로 받은'];
```

Comment:
요부분도 마찬가지로 상수 취급되면 좋을 것 같아용! ㅎㅎ

```suggestion
const ANY_TAG_ITEM_LIST = ['받고 싶어한', '많이 선물한', '위시로 받은'];

```